tla/consensus/Traceccfraft.tla

```diff
@@ -130,7 +130,11 @@ DropMessages ==
     /\ l \in 1..Len(TraceLog)
     /\ UNCHANGED <<reconfigurationVars, serverVars, candidateVars, leaderVars, logVars>>
     /\ UNCHANGED <<l, ts>>
-    /\ Network!DropMessages(logline.msg.state.node_id)
+    \* Only drop messages when processing message events
+    /\ \/ /\ "from_node_id" \in DOMAIN logline.msg
+          /\ Network!DropMessagesTo(logline.msg.state.node_id, logline.msg.from_node_id)
+       \/ /\ "from_node_id" \notin DOMAIN logline.msg
+          /\ UNCHANGED <<messageVars>>
```

Comment:
I'm not sure if this is the correct place to make this distinction? I _think_ we only want to drop messages (during trace validation) when we're trying to match a "message sending" step from the logs, where `from_node_id` is present? But I could be wrong.